scripts/show-stats-table.py

```diff
@@ -126,9 +126,9 @@ def get_active_cell(selected_cells, view_type):
             if not args.template:
                 return
             # nilearn or freeview rendering
-            fsdir= args.template.replace('$', str(subjects[temp['row']]))
+            fsdir = args.template.replace('sub-*/', 'sub-{}/'.format(subjects[temp['row']]))
```

Comment:
Why make it so strict as `sub-*`/? Can we not let a user specify whatever template with an `*` in it?

Reply:
Right now it is consistent with the substitution in `app.py`.